### PR TITLE
PSMDB 4.0 doesn't start the service after reboot and cosmic support

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: percona-packaging/scripts/psmdb_builder.sh [hunk 1]
```diff
@@ -239,10 +239,14 @@ set_compiler(){
...
             export CXX=/usr/bin/g++-6
     fi
-    if [ x"${DEBIAN}" = xartful -o x"${DEBIAN}" = xbionic ]; then
+    if [ x"${DEBIAN}" = xbionic ]; then
         export CC=/usr/bin/gcc-7
             export CXX=/usr/bin/g++-7
     fi
+    if [ x"${DEBIAN}" = xcosmic ]; then
+        export CC=/usr/bin/gcc-8
+            export CXX=/usr/bin/g++-8
```
Comment (tplavcic) [inline percona-packaging/scripts/psmdb_builder.sh]: maybe just fix indentation here and two lines above also

File: percona-packaging/scripts/psmdb_builder.sh [hunk 2]
```diff
@@ -747,7 +760,7 @@ RPM_RELEASE=1
 DEB_RELEASE=1
 REVISION=0
 BRANCH="v4.0"
-REPO="https://github.com/percona/percona-server-mongodb.git"
+REPO="https://github.com/EvgeniyPatlan/percona-server-mongodb.git"
 PSM_VER="4.0.4"
 PSM_RELEASE="1.1"
```
Comment (tplavcic) [inline percona-packaging/scripts/psmdb_builder.sh]: Should this be changed to just "1"?

File: percona-packaging/scripts/psmdb_builder.sh [hunk 3]
```diff
@@ -747,7 +760,7 @@ RPM_RELEASE=1
 DEB_RELEASE=1
 REVISION=0
 BRANCH="v4.0"
-REPO="https://github.com/percona/percona-server-mongodb.git"
+REPO="https://github.com/EvgeniyPatlan/percona-server-mongodb.git"
```
[the remark this answers is not in the document]
Comment (EvgeniyPatlan) [reply percona-packaging/scripts/psmdb_builder.sh]: fixed

File: percona-packaging/scripts/psmdb_builder.sh (hunk 1 above)
Comment (EvgeniyPatlan) [reply percona-packaging/scripts/psmdb_builder.sh]: fixed

File: percona-packaging/scripts/psmdb_builder.sh (hunk 2 above)
Comment (EvgeniyPatlan) [reply percona-packaging/scripts/psmdb_builder.sh]: fixed

**Pull request merged**